src/documentation/ptransform-style-guide.md

```diff
@@ -0,0 +1,488 @@
+---
+layout: default
+title: "PTransform Style Guide"
+permalink: /documentation/ptransform-style-guide/
+---
+
+# PTransform style guide
+
+## A style guide for writers of new reusable PTransform's
```

Comment:
A subtitle shouldn't be a header -- just italics in the regular size. See maturity model for an example.

+1 to adding one extra # to everything. FYI -- toc configuration is 2..6, see top-level configuration.